nutils/topology.py

```diff
@@ -609,38 +608,63 @@ def boundary(self):
     '''
```

Comment:
Should we use the opportunity to move this to ConnectedTopology?

Reply:
That requires a bit more work than just moving the properties. The `ConnectedTopology` takes a precomputed connectivity. The `SimplexTopology` does not define a `boundary` or `interfaces` itself, but subclassing from `ConnectedTopology` requires computing the connectivity in `__init__`. We could add a, say, `ConnectedTopologyBase` with an abstract property `connectivity`, move the `boundary` and `interfaces` properties to this class and let `SimplexTopology` and `ConnectedTopology` derive from `ConnectedTopologyBase`.

Reply:
I see, agreed. ConnectedTopologyBase seems to be the right approach but let's leave it for later.